js/upload.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+(function (DOM, EffectChange, Scale, EffectLevel) {
+  var makeOnMouseDown = DOM.Event.make.onMouseDown;
+
+  var uploadFileInput = document.querySelector('#upload-file');
+
+  var imageContainer = document.querySelector('.img-upload__overlay');
+  var uploadCancelButton = imageContainer.querySelector('#upload-cancel');
+
+  var scaleDownButton = imageContainer.querySelector('.scale__control--smaller');
+  var scaleUpButton = imageContainer.querySelector('.scale__control--bigger');
+
+  var effectLine = imageContainer.querySelector('.effect-level__line');
+  var effectPin = imageContainer.querySelector('.effect-level__pin');
+  var effectDepth = imageContainer.querySelector('.effect-level__depth');
+
+  var imageEffects = imageContainer.querySelector('.img-upload__effects');
+
+  var imageDescriptionInput = imageContainer.querySelector('.text__description');
+
+  var onChangeEffect = function (evt) {
+    EffectChange.control.select(evt.target.value);
+    EffectChange.render();
+    EffectLevel.reset();
+    EffectLevel.calc();
+  };
+
+  var onScaleUpClick = function () {
+    Scale.increase();
+    Scale.render();
+  };
+
+  var onScaleDownClick = function () {
+    Scale.decrease();
+    Scale.render();
+  };
+
+  var onEffectLineDown = function (evt) {
+    evt.preventDefault();
+    EffectLevel.dragged = false;
+
+    var onEffectLineUp = function () {
+      if (!EffectLevel.dragged && (evt.target !== effectPin)) {
+        effectPin.style.left = evt.offsetX + 'px';
+        effectDepth.style.width = evt.offsetX + 'px';
+        EffectLevel.calc();
+      }
+
+      effectLine.removeEventListener('mouseup', onEffectLineUp);
+    };
+
+    effectLine.addEventListener('mouseup', onEffectLineUp);
+  };
+
+  var onEffectPinMouseDown = function (evt) {
+    EffectLevel.Start.setX(evt);
+    EffectLevel.Start.setOffset();
+  };
+
+  var onEffectPinMouseMove = function (x, y, moveEvt) {
+    window.requestAnimationFrame(function () {
+      if (EffectLevel.isInRange.min(moveEvt) && EffectLevel.isInRange.max(moveEvt)) {
+        effectPin.style.left = effectPin.offsetLeft - x + 'px';
+        effectDepth.style.width = effectPin.offsetLeft - x + 'px';
+      } else if (!EffectLevel.isInRange.min(moveEvt)) {
+        effectPin.style.left = EffectLevel.Range.MIN;
+        effectDepth.style.width = EffectLevel.Range.MIN;
+      } else if (!EffectLevel.isInRange.max(moveEvt)) {
+        effectPin.style.left = EffectLevel.Range.MAX + 'px';
+        effectDepth.style.width = EffectLevel.Range.MAX + 'px';
+      }
+
+      EffectLevel.calc();
+      EffectLevel.dragged = true;
+    });
+  };
+
+  var onEffectPinMouseUp = function () {
+    effectPin.addEventListener('click', function (evt) {
+      evt.preventDefault();
+    }, {once: true});
+  };
+
+  function UploadPreview() {
```

Comment:
[`Д5`](https://up.htmlacademy.ru/javascript/17/criteries#d5)
Все функции объявлены единообразно

Функций-конструкторов это тоже касается :no_mouth:
```suggestion
  var UploadPreview = function () {
```